temp.c

```diff
@@ -19,35 +22,33 @@ void unos_history_celsius(float celsius, struct istorija memorija[], int *i) {
     memorija[*i].celsius = celsius;
   }
 }
```

Comment:
Sto se tice ove funkcije, ona je generalno dobra kao ideja. Ali nisam hteo to. Vi ovde u svaku funkciju prenosite ovo *i (vrlo lose ime, ne treba koristiti i za ovako nesto, vec neko ime koje nagovestava cemu ta promenljiva sluzi), sto nije dobro. Treba definisati globalnu promenljivu, iscitati malo vise o tome sta su globalne promenljive. To je promenljiva kojoj mozes pristupiti iz svakog mesta iz programa. Generalno su lose kao ideja, ali cemo kasnije videti sto. Za vas slucaj one su idealne. Treba ih definisati van svih funkcija, cak izvan i main() funkcije. 

Recimo ispod bloka gde su define naredbe treba staviti.
int trenutna_poz_unutar_memorija;

A zatim treba ukloniti sve ove argumente preko kojih koristite *i i tome slicno i onda direktno samo koristiti ovu promenljivu.